addons/sys_core/fnc_callExt.sqf

```diff
@@ -21,6 +21,18 @@
 
 params ["_command", "_params", ["_threaded", false], ["_callBack",{}], ["_callBackArgs",[]]];
 
+if (_threaded isEqualTo 2) exitWith {
```

Comment:
```suggestion
if (_threaded == 2) exitWith {
```
isn't this faster?

Reply:
isEqualTo is faster iirc

Reply:
For comparing integers too?

Reply:
the param is overloaded now, was bool, now `true/false/2`